client/exercises/links-and-buttons/button-vs-link/README.md

```diff
@@ -0,0 +1,11 @@
+### Button versus link
+
+Pentru că de multe ori se folosesc butoane în loc de link-uri sau invers, ne-am gândit la un exercițiu care să te ajute să înțelegi când îl folosim pe fiecare și de ce. 
+
+![button-vs-links-todos](./assets/added-to-cart.png)
+
+![button-vs-links-todos](./assets/button-vs-link.png)
```

Comment:
Why are there 2 pictures here? Should the user implement both?

I see you implemented the first one in the solution, great! I'd say keep that and drop the second picture.